src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -962,6 +964,19 @@ export class SortFilterBar
           font-weight: bold;
         }
 
+        /**
+         * Fix to not shift the sort-bar options when get selected
+         */
+        #desktop-sort-selector li button::before,
+        #desktop-sort-selector .dropdown-label::before {
+          display: block;
+          content: attr(title);
+          font-weight: bold;
+          height: 0;
+          overflow: hidden;
+          visibility: hidden;
+        }
+
```

Comment:
Nice elegant approach!

Minor suggestion: could we use an attribute other than `title` for this, so that we don't get tooltips when hovering the labels? Something like `data-content`?